Menu/GameShell/10_Settings/Time/myvars.py

```diff
@@ -0,0 +1,5 @@
+
+
```

Comment:
These newlines are not necessary.

Reply:
Yup, agreed. I just wanted to maintain the current style from the other files.